axelrod/tests/unit/test_filters.py

```diff
@@ -16,6 +16,14 @@ class TestStrategy(object):
             'makes_use_of': ['game', 'length']
         }
 
+    class TestStrategy2(object):
+        classifier = {
+            'stochastic': True,
+            'inspects_source': False,
+            'memory_depth': 'infinity',
```

Comment:
> I think I had to add the line to cope with one or two strategies that have this entry and were causing the API to fall over.

I don't think we have any strategies with that entry (or at least we shouldn't):

```python
>>> import axelrod as axl
>>> players = [s() for s in axl.strategies]
>>> mem_lengths = [p.classifier['memory_depth'] for p in players]
>>> set(mem_lengths)
 {0, 1, 2, 3, 4, 5, 6, 9, 10, 11, 12, 16, 20, 40, 200, inf}

```

Did you pergaps put this in so that the API could pass `infinity` to a filter? (Should that be done at the API level perhaps? So that the API converts `infinity` to a `float`?)